Sources/TensorFlow/Loss.swift

```diff
@@ -141,6 +141,26 @@ public func categoricalHingeLoss<Scalar: TensorFlowFloatingPoint>(
     return max(Tensor(0), negative - positive + Tensor(1))
 }
 
+// Helper function for `losCoshLoss(predicted:expected:)`.
+@differentiable
+fileprivate func logCosh<Scalar: TensorFlowFloatingPoint>(
+    x: Tensor<Scalar>
```

Comment:
The argument should not be labeled here.
```suggestion
    _ x: Tensor<Scalar>
```